### ISSUE TITLE
Support for Razer Huntsman TE

### ISSUE THREAD

**Issue by shofwan** [openrazer/openrazer#958]
### Device Info

* Name: Huntsman Tournament edition
* Product Number: RZ03-03080100
* Model Number: RZ03-0308

### Device Information
```
    lsusb -d 1532: -v
Bus 001 Device 002: ID 1532:0243 Razer USA, Ltd 
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 (Defined at Interface level)
  bDeviceSubClass         0 
  bDeviceProtocol         0 
  bMaxPacketSize0        64
  idVendor           0x1532 Razer USA, Ltd                                                                                                                                                                                                                                     
  idProduct          0x0243                                                                                                                                                                                                                                                    
  bcdDevice            2.00                                                                                                                                                                                                                                                    
  iManufacturer           1                                                                                                                                                                                                                                                    
  iProduct                2                                                                                                                                                                                                                                                    
  iSerial                 3                                                                                                                                                                                                                                                    
  bNumConfigurations      1                                                                                                                                                                                                                                                    
  Configuration Descriptor:                                                                                                                                                                                                                                                    
    bLength                 9                                                                                                                                                                                                                                                  
    bDescriptorType         2                                                                                                                                                                                                                                                  
    wTotalLength          109                                                                                                                                                                                                                                                  
    bNumInterfaces          4                                                                                                                                                                                                                                                  
    bConfigurationValue     1                                                                                                                                                                                                                                                  
    iConfiguration          0                                                                                                                                                                                                                                                  
    bmAttributes         0xa0                                                                                                                                                                                                                                                  
      (Bus Powered)                                                                                                                                                                                                                                                            
      Remote Wakeup                                                                                                                                                                                                                                                            
    MaxPower              500mA                                                                                                                                                                                                                                                
    Interface Descriptor:                                                                                                                                                                                                                                                      
      bLength                 9                                                                                                                                                                                                                                                
      bDescriptorType         4                                                                                                                                                                                                                                                
      bInterfaceNumber        0                                                                                                                                                                                                                                                
      bAlternateSetting       0                                                                                                                                                                                                                                                
      bNumEndpoints           1                                                                                                                                                                                                                                                
      bInterfaceClass         3 Human Interface Device                                                                                                                                                                                                                         
      bInterfaceSubClass      1 Boot Interface Subclass                                                                                                                                                                                                                        
      bInterfaceProtocol      1 Keyboard                                                                                                                                                                                                                                       
      iInterface              0                                                                                                                                                                                                                                                
        HID Device Descriptor:                                                                                                                                                                                                                                                 
          bLength                 9                                                                                                                                                                                                                                            
          bDescriptorType        33                                                                                                                                                                                                                                            
          bcdHID               1.11                                                                                                                                                                                                                                            
          bCountryCode            0 Not supported                                                                                                                                                                                                                              
          bNumDescriptors         1                                                                                                                                                                                                                                            
          bDescriptorType        34 Report                                                                                                                                                                                                                                     
          wDescriptorLength      61                                                                                                                                                                                                                                            
         Report Descriptors:                                                                                                                                                                                                                                                   
           ** UNAVAILABLE **                                                                                                                                                                                                                                                   
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 No Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     179
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0016  1x 22 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 No Subclass
      bInterfaceProtocol      2 Mouse
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      94
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        3
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 No Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              2 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     348
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x84  EP 4 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0002  1x 2 bytes
        bInterval               1

```

```
    ls -lah /dev/input/by-id/
total 0
drwxr-xr-x 2 root root 200 Nov 29 00:07 .
drwxr-xr-x 4 root root 660 Nov 29 00:07 ..
lrwxrwxrwx 1 root root  10 Nov 29 00:07 usb-CNFEH53M044040019062_Integrated_Webcam_HD_0001-event-if00 -> ../event15
lrwxrwxrwx 1 root root  10 Nov 29 00:07 usb-Logitech_USB_Receiver-if02-event-mouse -> ../event11
lrwxrwxrwx 1 root root   9 Nov 29 00:07 usb-Logitech_USB_Receiver-if02-mouse -> ../mouse2
lrwxrwxrwx 1 root root  10 Nov 29 00:07 usb-Razer_Razer_Huntsman_Tournament_Edition_00000000001A-event-if03 -> ../event10
lrwxrwxrwx 1 root root   9 Nov 29 00:07 usb-Razer_Razer_Huntsman_Tournament_Edition_00000000001A-event-kbd -> ../event6
lrwxrwxrwx 1 root root   9 Nov 29 00:07 usb-Razer_Razer_Huntsman_Tournament_Edition_00000000001A-if01-event-kbd -> ../event7
lrwxrwxrwx 1 root root   9 Nov 29 00:07 usb-Razer_Razer_Huntsman_Tournament_Edition_00000000001A-if02-event-mouse -> ../event9
lrwxrwxrwx 1 root root   9 Nov 29 00:07 usb-Razer_Razer_Huntsman_Tournament_Edition_00000000001A-if02-mouse -> ../mouse1
```

### Packet Captures

<Please see here as we probably need packet captures>
<https://github.com/openrazer/openrazer/wiki/Reverse-Engineering-USB-Protocol>

### Synapse Screenshots
<Screenshots of all available options as seen on Razer Synapse>

**Comment by ghost** [comment]
can confirm device not recolonised

**Comment by z3ntu** [comment]
Fixed with #1010
